apisix/cli/etcd.lua

```diff
@@ -241,6 +241,11 @@ function _M.init(env, args)
         util.die("the etcd cluster needs at least 50% and above healthy nodes\n")
     end
 
+    if not yaml_conf.apisix.enable_admin then
```

Comment:
it's a break change, should use a new field to do this

Reply:
Makes sense, let's keep it as a demo for those who want to use `read-only` DP. :)